plugins/lang/miscstats.txt

```diff
@@ -41,7 +41,8 @@ CT = CT
 CTS = CTS
 TERRORIST = TERRORIST
 TERRORISTS = TERRORISTS
-REMAINING = %d %s Remaining...
+REMAINING_ENEMY = Last enemy remaining...
```

Comment:
Never trust anyone!

Maybe it would be better if it's just "1 enemy remaining" or "%d enemy remaining" so it's remain the same with its plural version "%d enemies remaining".

Reply:
I wanted to aboid `%d' because it will be always 1, and it would be dumb to keep it. Also, it should give more freedom to translate. Like you could say either "1 enemy remaining", or someone would prefer "One enemy remaining".